surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/fixture/HelperAssertions.java

```diff
@@ -159,4 +164,10 @@ public static IntegrationTestSuiteResults parseReportList( List<ReportTestSuite>
             throw new RuntimeException( "Couldn't parse XML reports", e );
         }
     }
+
+    public static void assumeJavaVersion(JavaVersion version)
+    {
+        assumeThat( "java.specification.version: ",
```

Comment:
Hey Tibor,

do you mean to use a constant instead of the string in the message part of `assumeThat`? There is no constant for the property name in Commons Lang. But I could introduce one in `HelperAssertions` or we could extend SystemUtils, but then we have to wait for the next release of Commons Lang.